### DISCUSSION TITLE
distintos margenes de ganancia para grupo o categorias de productos en una misma lista de precios

### DISCUSSION THREAD

**Discussion by amarinozzi** [OdooAR/odoo-argentina-doc#8] [Q&A]
odoo13ce, como se puede tener distinto calculo de precio de venta de producto ( en base a categoría de producto u otra diferenciación) en una misma lista de precio?

**Comment by ibuioli** [comment]
Podés generar dos líneas para el mismo producto en la lista, supongamos que tenes el producto Cerveza y una la vendes a al precio de lista (100 por ejemplo) más el 30%. En la línea del producto podes tener el producto Cerveza con ese cálculo. Si además agregás otra línea en la lista de precio haciendo referencia al mismo producto pero que con la compra de 20 unidades o más aplica un 10% de descuento, entonces el sistema se va a comportar de otra manera. Es decir, salvo que ambas lineas con el mismo producto se pisen va a tomar la más cara.
